src/content/developers/docs/blocks/index.md

```diff
@@ -44,18 +44,20 @@ Proof of work means the following:
 
 ## What's in a block? {#block-anatomy}
 
-- Timestamp – the time when the block was mined.
-- Block number – the length of the blockchain in blocks.
-- Difficulty – the effort required to mine the block.
-- mixHash – a unique identifier for that block.
-- A parent hash – the unique identifier for the block that came before (this is how blocks are linked in a chain).
-- Transactions list – the transactions included in the block.
-- State root – the entire state of the system: account balances, contract storage, contract code and account nonces are inside.
-- Nonce – a hash that, when combined with the mixHash, proves that the block has gone through [proof of work](/developers/docs/consensus-mechanisms/pow/).
+- `timestamp` – the time when the block was mined.
+- `blockNumber` – the length of the blockchain in blocks.
+- `baseFeePerGas` - the fee per gas required for a transaction to be included in the block.
+- `burntFees` - the amount of ETH from base fees burnt by the block.
```

Comment:
This is not part of the block header, you can infer it (`baseFeePerGas * gasUsed`), but it isn't an actual field in the header.